### PR TITLE
Fix share link

### LINKED ISSUE #546: Share link has trailing %27
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

The link should be `https://www.who.int/covid-19-app` instead of `https://www.who.int/covid-19-app%27` as described in #291.

### PR DESCRIPTION
**Pull request by creativecreatorormaybenot**
## What does this PR accomplish?

Fixes #546.

## Did you add any dependencies?

No.

## How did you test the change?

- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by advayDev1** [comment]
Please pull out the pubspec lock file out of this PR.  Just fix the typo - thanks!

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 Done.

Why are we tracking Pubspec then?

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 Done. I needed to do it differently now because #550 was merged before this PR.

File: client/flutter/lib/generated/intl/messages_all.dart [hunk 1]
```diff
@@ -15,43 +15,43 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_ru_RU.dart' as messages_ru_ru;
-import 'messages_es_ES.dart' as messages_es_es;
-import 'messages_en_GB.dart' as messages_en_gb;
 import 'messages_ar.dart' as messages_ar;
-import 'messages_fr_FR.dart' as messages_fr_fr;
 import 'messages_en.dart' as messages_en;
+import 'messages_en_GB.dart' as messages_en_gb;
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_all.dart]: Unless this file has to be changed, please revert.

File: client/flutter/lib/generated/l10n.dart [hunk 2]
```diff
@@ -512,7 +512,7 @@ class AppLocalizationDelegate extends LocalizationsDelegate<S> {
 
   List<Locale> get supportedLocales {
     return const <Locale>[
-      Locale.fromSubtags(languageCode: 'ru', countryCode: 'RU'), Locale.fromSubtags(languageCode: 'es', countryCode: 'ES'), Locale.fromSubtags(languageCode: 'en', countryCode: 'GB'), Locale.fromSubtags(languageCode: 'ar'), Locale.fromSubtags(languageCode: 'fr', countryCode: 'FR'), Locale.fromSubtags(languageCode: 'en'), Locale.fromSubtags(languageCode: 'en', countryCode: 'US'), Locale.fromSubtags(languageCode: 'zh', countryCode: 'CN'),
+      Locale.fromSubtags(languageCode: 'ar'), Locale.fromSubtags(languageCode: 'en'), Locale.fromSubtags(languageCode: 'en', countryCode: 'GB'), Locale.fromSubtags(languageCode: 'en', countryCode: 'US'), Locale.fromSubtags(languageCode: 'es', countryCode: 'ES'), Locale.fromSubtags(languageCode: 'fr', countryCode: 'FR'), Locale.fromSubtags(languageCode: 'ru', countryCode: 'RU'), Locale.fromSubtags(languageCode: 'zh', countryCode: 'CN'),
```
Comment (advayDev1) [inline client/flutter/lib/generated/l10n.dart]: Unless this array has to be changed to fix an issue, please revert (but keep the L52 change ofc).  Don't want to mix general cleanup with a bugfix.

**Comment by advayDev1** [comment]
Sorry this is taking so long but we need to keep bugfix PRs scoped to just the bugfix to minimize the surface area needing to be retested.

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 I did not change this. This is what the Flutter Intl plugin did. I followed what @patniemeyer said [here](https://who-app.slack.com/archives/C0100FTK5FB/p1585447784259600).  

How should I change intl strings if this is not the way?

File: client/flutter/lib/generated/intl/messages_all.dart (hunk 1 above)
Comment (creativecreatorormaybenot) [reply client/flutter/lib/generated/intl/messages_all.dart]: This is done by the Flutter Intl plugin automatically.

**Comment by advayDev1** [comment]
@creativecreatorormaybenot no worries.  so the two changes I commented are just the plugin?  if so, please so state and I will LGTM.

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 To be sure, I will hard reset my branch and apply the changes again. There might be something else doing this (I did not touch it manually).

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 So yes, this is all done by the plugin. 

I only changed the single lines in the `l10n` directory and the plugin updated everything in `generated` automaitcally.

**Comment by advayDev1** [comment]
Great looks good from code.  Over to @sapte91 for loc.

**Comment by sapte91** [comment]
lgtm. @clementmouchet will take care of syncing any arb file updates with Lingo24.

**Comment by creativecreatorormaybenot** [comment]
@advayDev1 I think this PR needs your approval.

**Pull request merged**